content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,28 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.12 [2020-06-12]
+
+### Features
+
+- Make all `pkg` applications stateful through stacks.
+- Add ability to export a stack's existing resource state as a `pkg`.
+- Update influx `pkg` commands with improved usage and examples in long form.
```

Comment:
our docs say `pkg` was deprecated - does it make sense to say templates or something else...?

Reply:
that's true, we can rename this to `influx apply` command 🤔

Reply:
To be sure, would this be accurate:
Make all `influx apply` applications stateful through stacks.

Reply:
yah, that sounds right

Reply:
@jsteenb2 Would "as a `pkg`" translate to using `influx apply`?

Reply:
yah, the pkg commands have been split up a bit, so not all are going to be under `apply`. The new commands are all moving towards normalizing|standardizing on the name `templates`

Reply:
@jsteenb2 Makes sense, so to export a stack's existing resource state, would one use `influx export` (rather than`influx apply`)?

Reply:
Absolutely. Originally the export was buried under the `influx pkg` cmd as `influx pkg export`.  Now we can break that heirarchy a fair bit which is wonderful 👍